Sources/SwiftSyntax/Documentation.docc/ChangingSwiftSyntax.md

```diff
@@ -6,74 +6,18 @@ Discover the steps necessary to add, remove, and update syntax nodes
 
 The surface area of the entire Swift programming language is represented in the
 Swift Syntax API and its associated data structures. In order to keep these
-structures in sync, this project makes use of a code generation utility called
-[gyb][GYB] - short for "Generate Your Boilerplate". 
-
-## Introduction to GYB
-
-The gyb utility allows for [literate metaprogramming][LiterateProgramming] of 
-files in any human-editable format. The standard convention is to attach the
-`.gyb` suffix to a file. For example, this repository uses `.swift.gyb` files
-that generate `.swift` code. The metaprogramming language that gyb uses is
-python. Templating code is delimited by `%` and can be used inline with `${...}`
-blocks. 
-
-Here is the gyb definition for all of the cases of the <doc:TokenKind> enum.
-
-```swift
-public enum TokenKind: Hashable {
-  case eof
-% for token in SYNTAX_TOKENS:
-%   kind = token.swift_kind()
-%
-%   # Tokens that don't have a set text have an associated value that
-%   # contains their text.
-%   if not token.text:
-%     kind += '(String)'
-%   end
-  case ${kind}
-% end
-}
-```
-
-- Note: gyb files act as the source of truth in this repository. Always edit
-        gyb files instead of the corresponding Swift files, or your changes will
-        be overwritten the next time those files are regenerated.
+structures in sync, this project makes generates the code using the Swift package 
+`CodeGeneration`, located in the root of this repository. 
 
 ## Regenerating Files
 
-While working on gyb files, it can be helpful to regenerate the files in this
-repository to see the effects of local changes. SwiftSyntax comes with a script
-called `build-script.py` that automates the process of regenerating, validating,
-and building all of the files in this repository. 
-
-To invoke `build-script.py` to regenerate source code, run
-
-```
-$ ./build-script.py generate-source-code --toolchain /path/to/toolchain/usr
-```
-
-where `/path/to/toolchain` can point either to the `XcodeDefault.xctoolchain`
-installed with Xcode, or to a [toolchain downloaded from swift.org][SwiftToolchains].
-
-- Note: To see the raw commands that `build-script.py` is running, use the `-v`
-        flag. This can be used to dump the command(s) necessary to regenerate
-        one file at a time.
-
-This script also comes with a verification command that is run by Swift's
-CI/CD systems to ensure that you regenerate files before they are checked into
-the version control system. 
-
-To invoke `build-script.py` in verification mode, run
-
-```
-$ ./build-script.py verify-source-code --toolchain /path/to/toolchain/usr
-```
+To re-generate the files after changing `CodeGeneration` run the `generate-swiftsyntax` 
+target of `CodeGeneration` and pass `path/to/swift-syntax/Sources` as the argument.
```

Comment:
`generate-source-code` still exists and IMO is easier than having to run `generate-swiftsyntax`. But if you'd prefer the latter, it'd be nice to spell it out completely:
```
swift run --package-path CodeGeneration generate-swift syntax Sources
```
(not sure if we handle the relative path there)

Reply:
I prefer to tell people to just run `generate-swiftsyntax` now. I’d like to reduce the number of uses for `build-script.py`. Maybe one day we can even get rid of that last remaining Python script. 

I added the command though, that’s a good suggestion.

Reply:
Do we need to keep `generate-source-code`?